docs/content/_snippets/deprecated_methods_note.mdx

```diff
@@ -0,0 +1,3 @@
+:::note
```

Comment:
This is just a comment about how this file is named very generic but the content is very specific, by the name it looks to be a generic deprecation note but is just for 2 very specific methods. In the future if you need more deprecations this file might need a rename. This is just my inner coder speaking 🌷